app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,9 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all
+  end
+
+  def show
```

Comment:
We don't need a `show` action in our `Ingredients` controller. We never visit an ingredients show page.